.github/workflows/e2e_android.yml

```diff
@@ -10,6 +10,10 @@ on:
       - 'dependabot/**'
   workflow_dispatch:
     inputs:
+      tests_to_test:
```

Comment:
Consider calling this "tests to run" instead of "tests to test", I think it's a bit clearer. Same for the other uses below

Reply:
Changed to tests_to_run